peachjam/resources.py

```diff
@@ -129,12 +132,26 @@ def before_import_row(self, row, **kwargs):
         DocumentNature.objects.get_or_create(name=row["nature"])
 
 
+class JudgesWidget(ManyToManyWidget):
+    def clean(self, value, row=None, *args, **kwargs):
+        judges = list(map(" ".join, list(map(str.split, value.split(self.separator)))))

```

Comment:
I'm not clear what this is doing. Can you add a comment? Also, list comprehensions are generally easier to read and more pythonic than using `map`.

Reply:
This is supposed to strip white space around and between the judges name to avoid the duplication issue we encountered. I have changed it to a list comprehension.